include/swift/Serialization/SerializedModuleLoader.h

```diff
@@ -42,8 +42,10 @@ class SerializedModuleLoaderBase : public ModuleLoader {
 protected:
   ASTContext &Ctx;
   ModuleLoadingMode LoadMode;
+  ArrayRef<std::string> PreferInterfaceForModules;
```

Comment:
Sorry I missed this the first time around. This doesn't belong in SerializedModuleLoaderBase; please move it up to ParseableInterfaceModuleLoader.